recipes/android-ndk/all/test_package/conanfile.py

```diff
@@ -4,7 +4,7 @@ class TestPackgeConan(ConanFile):
     settings = "os", "arch"
 
     def build(self):
-        pass #nothing to do, not warnings please
+        pass
```

Comment:
Removing the comment might trigger someone to remove the pass, and the warnings will be back